pkg/metrics/metrics.go

```diff
@@ -73,9 +74,10 @@ func (rw *responseWriter) WriteHeader(code int) {
 func PrometheusMiddleware(ph http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 
-		metricRequestsTotal.WithLabelValues().Inc()
-
 		path, _ := mux.CurrentRoute(r).GetPathTemplate()
+		if path != handlers.LivezPath && path != handlers.ReadyzPath {
+			metricRequestsTotal.WithLabelValues().Inc()
+		}
 		metricRequestsByService.WithLabelValues(path).Inc()
```

Comment:
I think we should put this in the if condition as well?
Since collect the metric for the livez and readyz is also useless.

Reply:
Thanks for the review, I also had a similar thought that we might not want to record the path requests to livez & readyz. But initially it posed as a good testing condition for clarification. 
I will make this change.

Reply:
Made this change and tested it against a cluster.